backend/src/test/java/com/phaete/backend/forage/service/ForageMapItemServiceTest.java

```diff
@@ -16,67 +16,69 @@ class ForageMapItemServiceTest {
 	private final ForageMapItemRepository forageMapItemRepository = mock(ForageMapItemRepository.class);
 	private final IdService idService = mock(IdService.class);
 
+	ForageMapItemService forageMapItemService = new ForageMapItemService(
+			forageMapItemRepository,
+			new ConverterService(idService)
+	);
+
+	ForageMapItemDTO expectedForageMapItemDTO = new ForageMapItemDTO(
+			new ForageWikiItem(
+					"1",
+					"Apple Tree",
+					ForageCategory.FRUIT,
+					ForageSource.TREE,
+					"Apple Tree",
+					ForageSeason.FALL,
+					List.of("test")
+			),
+			new CustomMarker(
+					"1",
+					new double[] {0.0, 0.0},
+					"",
+					new int[] {0, 0},
+					new int[] {0, 0},
+					new int[] {0, 0},
+					""
+			),
+			ForageQuantity.ABUNDANT,
+			ForageQuality.EXCELLENT,
+			"never",
+			"notes"
+	);
+
+	ForageMapItem expectedForageMapItem = new ForageMapItem(
+		"1",
+			new ForageWikiItem(
+					"1",
+					"Apple Tree",
+					ForageCategory.FRUIT,
+					ForageSource.TREE,
+					"Apple Tree",
+					ForageSeason.FALL,
+					List.of("test")
+			),
+			new CustomMarker(
+					"1",
+					new double[] {0.0, 0.0},
+					"",
+					new int[] {0, 0},
+					new int[] {0, 0},
+					new int[] {0, 0},
+					""
+			),
+			ForageQuantity.ABUNDANT,
+			ForageQuality.EXCELLENT,
+			"never",
+			"notes"
+	);
+
 	@Test
 	void createForageMapItem_expectDTO_onSuccess() {
-		ForageMapItemDTO expectedForageMapItemDTO = new ForageMapItemDTO(
-				new ForageWikiItem(
-						"1",
-						"Apple Tree",
-						ForageCategory.FRUIT,
-						ForageSource.TREE,
-						"Apple Tree",
-						ForageSeason.FALL,
-						List.of("test")
-				),
-				new CustomMarker(
-						"1",
-						new double[] {0.0, 0.0},
-						"",
-						new int[] {0, 0},
-						new int[] {0, 0},
-						new int[] {0, 0},
-						""
-				),
-				ForageQuantity.ABUNDANT,
-				ForageQuality.EXCELLENT,
-				"never",
-				"notes"
-		);
-
 		when(forageMapItemRepository.save(any(ForageMapItem.class))).thenReturn(
```

Comment:
Now as your code has become clearer, I see that you use "any" here.
That is not necessary as you are mocking your IdService and can use a mocked id.